lib/src/test/java/org/team100/lib/localization/TestCameraAngles.java

```diff
@@ -10,11 +10,12 @@ class TestCameraAngles extends Fixtured {
     @Test
     void testCameraAngles() {
         {
-            CameraAngles camera = new CameraAngles(30, 67.5, 50, 832, 616, 1, 0,0);
-               assertEquals(0,camera.getY(416,200),0.0001);
-               assertEquals(-0.7,camera.getX(0),0.001);
-               assertEquals(416,camera.getInverseY(1,0),0.001);
-               assertEquals(123.2,camera.getInverseX(1),0.001);
+            CameraAngles camera = new CameraAngles(Math.toRadians(30), 1, 0,0);
```

Comment:
i think you should make a few more very simple test cases here to be sure you have the transformation logic correct.  like if the camera returns a target in the center (i.e. yaw = 0, pitch = 0) and if the camera offset is something very easy to calculate like 1 meter up, 45 degrees down, do you get the correct result (1 meter ahead)?